### DISCUSSION TITLE
[Feature] Add option to only see the favourites of an album

### DISCUSSION THREAD

**Discussion by lyz-code** [immich-app/immich#7719] [Feature Request]
### The feature

When I want to show the content of an album to some friends I'd like to show them only the favourited elements. Right now if you click on an album you can only see all of them.

The solution so far is to see all the favourites and search the date where the album content is (in case of a trip for example) and browse from there. It's not the ideal solution though

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by mglaessel** [comment]
Oh yes I agree so much. Very important and not much effort to implement.

**Comment by bo0tzz** [comment]
#12258
